mq/plugins/auditdFixup.py

```diff
@@ -114,7 +114,4 @@ def onMessage(self, message, metadata):
         if 'category' not in message:
             message['category'] = 'auditd'
 
-        # set doctype
-        metadata['doc_type'] = 'auditd'
```

Comment:
I think we still want to specifically set the type here to auditd. We don't have a reliable way to tag these events as auditd events, because while we're setting the category to 'auditd', it's an if statement, so if the incoming message already has that field set, we won't set the category as auditd.

I think we may want to either remove the about if statement to check if category exists, or we set the type to auditd and shore up the alerts that use category, to use 'type'.

Reply:
We set message's `type` as 'auditd` here https://github.com/mozilla/MozDef/pull/1183/files#diff-a4c28c693516dbfaaf05ad3a682dfbbcR20  so whatever we decide, we'll want to sync that up as well.

Reply:
I think if the category is auditd it should be fine, unless we want to further specify the type of auditd event, (syscall, exec, etc)

Reply:
The problem is that we might not set the category to autitd if the incoming message already has a category set, because that assignment is based on a conditional at https://github.com/mozilla/MozDef/blob/ff20881548201c7aafbdde5e7b84760de1502d87/mq/plugins/auditdFixup.py#L114

So I'm saying that we might want to explicitly set type to auditd here just so we reliably have a way to "tag" incoming auditd events.

Thoughts?

Reply:
Is there a way for us to reliably know that "type" isn't already set to something we may find valuable that may need to be cast into another field? Or was type only added to this because maybe it was like that in the original template?

Reply:
I'm not really sure, but my guess would be that it was added like that in the original template.

Reply:
I just looked up and didn't see any auditd events that already have `type` set, so I think we should be fine?

Reply:
Sounds good!